pipigraph.py

```diff
@@ -1,29 +1,76 @@
 import time
-
 from winsound import Beep
 import serial
+import random
 
 
 class PipiGraph:
 
-    def __init__(self):
+    def __init__(self, values):
         self.device = self.create_connection()
+        self.data = []
+        for x in values:
+            self.data.append((x[0], x[1][0]))
```

Comment:
toto moc nedava zmysel kedze sa len zoberu hodnoty a prekopiruju sa do ineho zoznamu

Reply:
zmenila som to uz sa to nekopiruje